test/link.test.js

```diff
@@ -103,3 +103,15 @@ test("pass through attributes", () => {
   expect(vnode.attributes.href).toEqual("/path")
   expect(vnode.attributes.pass).toEqual("through")
 })
+
+test('custom onclick handler', () => {
+  const event = {
+    preventDefault: jest.fn(),
```

Comment:
@frenzzy Could you implement this without jest.fn()?

Reply:
Why?

Reply:
I want to avoid it for now.